src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -355,6 +376,156 @@
     </StringProperty.Metadata>
   </StringProperty>
 
+  <!-- TODO: Shortlink for help URL -->
+  <!--Description="Enable modern visual styling for control rendering using the latest Control Library (COMCTL)."-->
+  <BoolProperty Name="EnableVisualStyles"
+                DisplayName="Windows XP visual style"
+                Description="Enable the use the Windows XP styles for the application."
+                HelpUrl="https://docs.microsoft.com/dotnet/api/microsoft.visualbasic.applicationservices.windowsformsapplicationbase.enablevisualstyles"
+                Category="ApplicationFramework">
+    <BoolProperty.Metadata>
+      <NameValuePair Name="DependsOn" Value="Application::UseApplicationFramework" />
+      <NameValuePair Name="VisibilityCondition">
+        <NameValuePair.Value>
+          (has-evaluated-value "Application" "UseApplicationFramework" true)
```

Comment:
Please test the following:

1. Set output type to WinExe
2. Check "Use application framework"
3. Verify this property is visible
4. Change from WinExe to Library
5. Verify this property is hidden

I suspect this property may remain visible in the last of the above steps. I don't think we set `UseApplicationFramework` to `false` when making the `OutputType` something other than `WinExe`.

It would be possible to do that with interception, but then there'd be a delay between changing that drop-down and the properties being hidden.

Instead it might be better to `and` this (and other VB properties) with `(has-evaluated-value "Application" "OutputType" "WinExe")`.